### DISCUSSION TITLE
SVG 태그 관련 접근성 정보 공유

### DISCUSSION THREAD

**Discussion by chaewon0128** [marketkarly-team/marketkarly-client#28] [General]
SVG 태그 사용에 관한 접근성을 잘 정리해둔 사이트 입니다 (야무위키✨)
마크업시 참고 하시면 좋을 것 같습니다.

[SVG접근성](https://a11y.gitbook.io/graphics-aria/svg-graphics/svg-a11y)

**Comment by jaesukpark77** [comment]
해당 부분은 내일 혹은 목요일에 의논해 보면 좋을듯싶습니다!!!😀😀😀
